src/Test/test_linear.jl

```diff
@@ -190,7 +174,7 @@ function test_linear_integration(
     else
         MOI.delete(model, c)
         cf = MOI.ScalarAffineFunction{T}(
-            MOI.ScalarAffineTerm{T}.([one(T), one(T), one(T)], v),
+            MOI.ScalarAffineTerm{T}.(one(T), v),
```

Comment:
`T(1)`
are we standardizing?

Reply:
I switched to `T(1)` and `T(0)`. We assume you can go `T(3)`, so we might as well go all-in.